www-misc/buku/buku-4.1.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6} )
+
+inherit bash-completion-r1 distutils-r1
+
+DESCRIPTION="Powerful command-line bookmark manager"
+HOMEPAGE="https://github.com/jarun/Buku"
+
+SRC_URI="https://github.com/jarun/Buku/files/1319933/bookmarks.zip"
```

Comment:
This is a very bad and collision-prone filename. Please rename it to include 'buku' prefix, and some version indicator.

Reply:
Also, if you need it only for tests, `test? ( ... )`.